kolibri/core/templatetags/kolibri_tags.py

```diff
@@ -136,6 +137,9 @@ def kolibri_language_globals(context):
             common_css_file=common_file,
             subset_css_file=subset_file,
             full_css_file=full_file,
+            # Temporary cache busting strategy.
+            # Would be better to use ManifestStaticFilesStorage
```

Comment:
Using this would mess up dynamic fetching of files by webpack :/

Reply:
So, until we can work that out, this is the right approach.

Reply:
unfortunately it doesn't cover the woff files referenced _inside_ of the css files. Not a huge deal because they don't change as often, but would still be good to address

Reply:
I wonder if it's possible to configure multiple django static file stores